.travis.yml

```diff
@@ -29,52 +30,23 @@ env:
   global:
     - LUAROCKS=2.4.3
     - OPENSSL=1.0.2n
-    - CASSANDRA=3.9
     - OPENRESTY_BASE=1.13.6.2
     - OPENRESTY_LATEST=1.13.6.2
     - OPENRESTY=$OPENRESTY_BASE
     - DOWNLOAD_CACHE=$HOME/download-cache
     - INSTALL_CACHE=$HOME/install-cache
     - JOBS=2
   matrix:
-    - TEST_SUITE=pdk
-    - TEST_SUITE=lint
-    - TEST_SUITE=unit
-    - TEST_SUITE=integration
-      OPENRESTY=$OPENRESTY_LATEST
-    - TEST_SUITE=plugins
-      OPENRESTY=$OPENRESTY_LATEST
-    - TEST_SUITE=integration
-      OPENRESTY=$OPENRESTY_BASE
-      CASSANDRA=2.2.12
-    - TEST_SUITE=plugins
-      OPENRESTY=$OPENRESTY_BASE
-      CASSANDRA=2.2.12
-    - TEST_SUITE=old-unit
-    - TEST_SUITE=old-integration
-      OPENRESTY=$OPENRESTY_BASE
-    - TEST_SUITE=old-plugins
-      OPENRESTY=$OPENRESTY_BASE
+    - KONG_TEST_DATABASE=postgres KONG_TEST_PG_USER=postgres KONG_TEST_PG_DATABASE=travis
+    - KONG_TEST_DATABASE=cassandra KONG_TEST_CASSANDRA_KEYSPACE=kong_tests CASSANDRA=2.2 KONG_TEST_DB_UPDATE_PROPAGATION=1
+    - KONG_TEST_DATABASE=cassandra KONG_TEST_CASSANDRA_KEYSPACE=kong_tests CASSANDRA=3.9 KONG_TEST_DB_UPDATE_PROPAGATION=1
```

Comment:
When a range of OpenResty versions is supported (vs. a single one as of this moment), we'll need to add a few jobs like so:
```
- OPENRESTY=$OPENRESTY_LATEST KONG_TEST_DATABASE=postgres KONG_TEST_PG_USER=postgres KONG_TEST_PG_DATABASE=travis
- OPENRESTY=$OPENRESTY_LATEST KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 KONG_TEST_DB_UPDATE_PROPAGATION=1
```

Does it make sense then to move `KONG_TEST_PG_USER=postgres KONG_TEST_PG_DATABASE=travis` to the `global` entry? Maybe `KONG_TEST_DB_UPDATE_PROPAGATION` could also be conditionally set in the `.ci/setup_env.sh` script (inside of the `if cassandra then` branch). Reasoning here is to try to keep the cognitive load and duplication in the matrix as low as possible.

This way, we would have:
```
- KONG_TEST_DATABASE=postgres
- KONG_TEST_DATABASE=postgres OPENRESTY=$OPENRESTY_LATEST 
- KONG_TEST_DATABASE=cassandra CASSANDRA=2.2
- KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 OPENRESTY=$OPENRESTY_LATEST 
```